BedrockServer.cpp

```diff
@@ -380,7 +380,7 @@ void BedrockServer::sync()
 
             if (command->shouldPostProcess() && command->response.methodLine == "200 OK") {
                 // PostProcess if the command should run postProcess, and there have been no errors thrown thus far.
-                core.postProcessCommand(command);
+                core.postProcessCommand(command, false);
```

Comment:
Special attention here. I think here were are running in the `syncThread`, correct? Not sure if this time needs to be considered as time spent in the `blockingCommit` thread, I guess not.

Reply:
It's not the blocking commit thread.